definitions.h

```diff
@@ -295,13 +624,13 @@ class FragmentShader
 };
 
 // Example of a vertex shader
-void DefaultVertShader(Vertex & vertOut, Attributes & attrOut, const Vertex & vertIn, const Attributes & vertAttr, const Attributes & uniforms)
+void myVertexShader(Vertex & vertOut, Attributes & attrOut, const Vertex & vertIn, const Attributes & vertAttr, const Attributes & uniforms)
 {
-    // Nothing happens with this vertex, attribute
-    vertOut = vertIn;
+    vertOut = uniforms.transforms * vertIn;

```

Comment:
I see how simple it is, when you're in the VertexShader, to transform the vertex.  Good job on your implementation.